keyboards/handwired/dactyl_manuform/eeprom-lefthand.eep

```diff
@@ -0,0 +1,2 @@
+:0F000000000000000000000000000000000001F0
```

Comment:
The EEPROM files are the same as the lets split's.   Any reason to include them here, rather than use those?

Reply:
I guess the average user won't understand that.
for others it will be easier to apply it directly than look for it in other
keyboards folders and anyhow it's just 100 bytes so no harm done.

Reply:
But i can drop that if it's nuisance.